### ISSUE TITLE
Bug in plot.netdx when grid=TRUE

### ISSUE THREAD

**Issue by sgoodreau** [EpiModel/EpiModel#651]
It appears that if `type="formation"` and `plots.joined=TRUE` and `grid=TRUE` and one's plotting device is empty, then `grid()` gets called before the plot is set up, and an error is returned. This may occur elsewhere as well, so a thorough check should be done.

To see the issue, clear your plot window's history, and then run:

```
est <- netest(nw=network_initialize(n = 100),
              formation = ~edges,
              target.stats=50, 
              coef.diss = dissolution_coefs(~offset(edges), 50))
dx <- netdx(est, nsims = 3, nsteps = 100)
plot(dx, grid = TRUE)
```

**Comment by smjenness** [comment]
@sgoodreau : thanks for finding. Since you are in the weeds of this code now, would you be able to adress? I imagine it would be a relatively simple fix. Thanks!

**Comment by sgoodreau** [comment]
on it

**Comment by sgoodreau** [comment]
Resolved with fa662cfe19161392c971b657c43ec55221e7e822
